config/manage_container/manage1/.env

```
@@ -2,7 +2,7 @@ PARTY_ID=1
 N_PARTIES=3
 MANAGE=http://localhost:50011
 COMPUTATION=http://computation_container1:50010
-PARTY_LIST1=http://0.0.0.0:50010
-PARTY_LIST2=http://manage_container2:50010
-PARTY_LIST3=http://manage_container3:50010
+PARTY_LIST1=http://0.0.0.0:51011
+PARTY_LIST2=http://manage_envoy_container2:50010
+PARTY_LIST3=http://manage_envoy_container3:50010
```

Is it possible to unify ports like CC and CC-envoy?
It can be confusing when specifying ports in `.env` files.

```suggestion
PARTY_LIST1=http://0.0.0.0:51011
PARTY_LIST2=http://manage_envoy_container2:51011
PARTY_LIST3=http://manage_envoy_container3:51011
```

or

```suggestion
PARTY_LIST1=http://0.0.0.0:50010
PARTY_LIST2=http://manage_envoy_container2:50010
PARTY_LIST3=http://manage_envoy_container3:50010
```

---

@arukuka 
[PORT_FOR_JOB](https://github.com/acompany-develop/QuickMPC/blob/main/config/computation_container/compute1/.env#:~:text=http%3A//beaver_triple_service%3A50020-,PORT_FOR_JOB%3D51020,-PARTY_LIST1%3Dhttp%3A//computation_container1) is used both as the destination port and the listen port.
Therefore, the ports of CC and CC_envoy had to be the same

Should we unify the mc as well?

---

@KotaTakahashi9320 
I checked the internal documents.  
It's not the scope of this PR, but I would like to fix either CC or the documentation.